src/components/organisms/Slider/SlideItemContent.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// T Y P E S
+import { CardAuthorProps, CardAuthor } from './CardAuthor'
+
+// I N T E R F A C E
+export interface SlideItemContentProps {
```

Comment:
Same here extend `DefaultComponentProps` and handle `style` prop

Reply:
also removed because it's part of business logic